src/tlo/methods/healthburden.py

```diff
@@ -116,12 +119,16 @@ def initialise_simulation(self, sim):
         self.process_causes_of_disability()
         self.process_causes_of_dalys()
 
-        # 4) Launch the DALY Logger to run every month, starting with the end of the first month of simulation
-        sim.schedule_event(Get_Current_DALYS(self), sim.date + DateOffset(months=1))
-
+        # 4) Launch the DALY and Prevalence Logger to run every month, starting with the end of the first month of simulation
         # 5) Schedule `Healthburden_WriteToLog` that will write to log annually
-        last_day_of_the_year = Date(sim.date.year, 12, 31)
-        sim.schedule_event(Healthburden_WriteToLog(self), last_day_of_the_year)
+        sim.schedule_event(Get_Current_DALYS(self), sim.date + DateOffset(months=1))
+        if self.parameters['test']:
+            sim.schedule_event(Get_Current_Prevalence(self), sim.date + DateOffset(months=1))
```

Comment:
could we make a choice here for the frequency of logging - if we are interested in very rapidly developing/resolving conditions we could set to daily logger, for broader analyses we could set to annual!?

Reply:
I have included a parameter now that allows us to set the time of logging as either daily, monthly, or yearly